e2e_tests/tests/command/test_notebook.py

```diff
@@ -8,7 +10,7 @@
 def test_basic_notebook_start_and_kill() -> None:
     with cmd.interactive_command("notebook", "start") as notebook:
         for line in notebook.stdout:
-            if "Jupyter Notebook is running" in line:
+            if re.match(".*Jupyter Notebook .*is running at.*", line) is not None:
```

Comment:
non-blocking: The trailing `.*` is unnecessary, and (separately) the leading one could be removed by switching to `search` instead of `match`.

Reply:
done.